src/MUIDataTable.js

```diff
@@ -358,6 +366,12 @@ class MUIDataTable extends React.Component {
         );
       }
     });
+
+    if (this.options.disableToolbarSelect === true) {
```

Comment:
We should also add a deprecation notice for disableToolbarSelect === false.

Reply:
getDefaultOptions() will set disableToolbarSelect before handleOptionDeprecation() is called. This means that the deprecation warning would always be printed, even if the option is not set by the user.

I would like to avoid the rewrite necessary for that.

Reply:
That's a good point, but it should be done regardless. However, I can do that separately, so no worries on your end.